core/cloudflow-akka/src/main/scala/cloudflow/akkastream/persistence/AkkaPersistenceConfigurator.scala

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cloudflow.akkastream.persistence
+
+import java.nio.file.{ Files, Paths }
+
+import com.typesafe.config.{ Config, ConfigFactory }
+import net.ceedubs.ficus.Ficus._
+import org.slf4j.LoggerFactory
+
+import scala.collection.JavaConverters._
+
+object AkkaPersistenceConfigurator {
```

Comment:
This at first glance seems to be the kind of thing that should land in cli and config support.

Reply:
Sory, this is part of my experiments, will remove shortly